docs/source/plugin/bot.rst

```diff
@@ -392,41 +392,50 @@ As mentionned earlier, Sopel relies on a Python built-in module:
     datetime.datetime(2021, 7, 26, 16, 7, 16, 496404)
 
 As you can see at the moment of writing this documentation, there was a 2h
-offset between the local time and UTC. To properly manage timezone and UTC
-offset, it is best to rely on ``pytz``, and work with UTC only::
+offset between the local time and UTC. To properly manage timezones and UTC
+offsets, it is best to rely on ``pytz``, and work with UTC only::
 
     >>> import pytz
     >>> pytz.UTC.localize(datetime.datetime.utcnow())
     datetime.datetime(2021, 7, 26, 16, 7, 19, 321828, tzinfo=<UTC>)
 
-This way, you'll always have an aware datetime that doesn't depend on your
-local time to work with (store, compare, manipulate, etc.) and when you need it
-you will convert it to the proper timezone.
+This way, you'll always have an aware datetime to work with (store, compare,
+manipulate, etc.) that doesn't depend on your local time, and you'll only
+convert it to the proper timezone when you need to.
 
 .. note::
 
-    You should **always** work with aware datetime, and it's easier to always
-    work with UTC+0 datetime as input and storage then to convert into another
-    timezone when displaying time to a user.
+    You should **always** work with aware datetime objects. It's also easier to
+    always work with UTC+0 datetime as input and in storage, then convert to
+    another timezone when displaying time to a user.
 
 Getting the timezone
 --------------------
 
-When using ``pytz``, getting a timezone is straightforward (as long as you know
-the `IANA name of said timezone`__)::
+Sopel uses ``pytz`` to handle timezones to manipulate aware datetimes, i.e.
+datetimes with a timezone information such as the UTC Offset and DST
+(`Daylight Saving Time`__). When using this library, getting a timezone
+is straightforward (as long as you know the `IANA name of said timezone`__)::
 
     >>> tz = pytz.timezone('Europe/Paris')
+
+Then you can get an UTC datetime::
```

Comment:
```suggestion
Then you can get a UTC datetime::
```

"an" if UTC was read as "uhtk", sure. But most people are going to read it as an initialism and say "you tee see", so it'll be "a" in practice.

Don't you love English? 🙃

Reply:
No, I don't.